doc/source/changes/version_0_29.rst.inc

```diff
@@ -1,7 +1,8 @@
 ﻿Syntax changes
 --------------
 
-* new syntax
+* LArray.insert pos and axis arguments are deprecated because those were only useful for very specific cases and those
+  can easily be rewritten by using an indices group for the "before" argument instead (closes :issue:`652`).
```

Comment:
I'm not sure that users know what an "indices group" is. Maybe add something like (axis.i[pos]) after "an indices group".

Reply:
ok